src/containers/Search/index.tsx

```diff
@@ -209,19 +210,22 @@ const Search = (homeProps: IHomeProps) => {
   }, [])
 
   return (
-    <Container sx={styles.container} maxWidth={false}>
-      <Modal open={openModal} setOpen={setOpenModal} selectedClaim={selectedClaim} />
-      <NewClaim
-        open={openNewClaim}
-        setOpen={setOpenNewClaim}
-        selectedClaim={selectedClaim}
-        setLoading={setLoading}
-        setSnackbarMessage={setSnackbarMessage}
-        toggleSnackbar={toggleSnackbar}
-      />
-
-      <Box ref={ref} sx={styles.cy} />
-    </Container>
+    <Box>
+      {search ? null : <FeedClaim />}
```

Comment:
Consider writing this line like below: 
`{!search && <FeedClaim />}`

Reply:
i made it @ArnobChowdhury